source/main.cpp

```diff
@@ -171,6 +186,7 @@ extern "C" void recognizeFaces(unsigned char* sourceImageData, int rows, int col
                                   cv::Point2f { normedLandmarks[2], normedLandmarks[3] } };
                 auto rightEye = { cv::Point2f { normedLandmarks[4], normedLandmarks[5] },
                                    cv::Point2f { normedLandmarks[6], normedLandmarks[7] } };
+		beforeAlignFaces.push_back(detectedFaces[i]);
```

Comment:
I think we do not need an additional container for the detected faces and can use just detectedFaces vector, which you have made global